news/2020-11-29-a-labour-of-love.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "A Labour of Love"
+date: 2020-11-29 08:00:00 +0000
+---
+
+Are you a mapper? A storyboarder? Or maybe a talented music producer? Show your love for an osu! community favorite in this multi-layered contest!
+
+![](/wiki/shared/news/banners/alol_cover.jpg)
+
+Everyone knows [**Kuba Oms - My Love**](https://osu.ppy.sh/beatmapsets/163112). Put 100 osu! players in a room and you'd be safe betting that half of them could sing its chorus. The beatmap has nearly 70 *million* plays and is a source of nostalgia for generations of us.
+
+We paired with *Kuba Oms* to set up a contest celebrating this timeless classic — a contest more ambitious than any we've hosted before!
+
+## Teams
+
+Lots to go through. Let's keep it short.
+
+**This is a team-based contest.** Users with different skillsets must collaborate to create a *full beatmap experience*, including:
+
+- Remix of the song **Kuba Oms - My Love** (stems provided)
+- Single-difficulty beatmap (osu! only)
+- Storyboard or custom video
+
+**Submit your team name and member list to [this form](https://docs.google.com/forms/d/e/1FAIpQLScjtVsUWIArvf--pKA0RLiCdXuO1_wO2Va2ICjenEcz9ZqI5Q/viewform).** Once confirmed, you will receive the the stems for *My Love* and your team will be documented on the [A Labour of Love contest wiki page](/wiki/Contests/A_Labour_of_Love_Contest).
```

Comment:
may want to provide an alternative way to submit (contacting u?) cuz china + google